h2/src/main/org/h2/message/DbException.java

```diff
@@ -342,6 +341,42 @@ public static DbException convertIOException(IOException e, String message) {
         return get(ErrorCode.IO_EXCEPTION_2, e, e.toString(), message);
     }
 
+    /**
+     * Gets the SQL exception object for a specific error code.
+     *
+     * @param errorCode the error code
+     * @return the SQLException object
+     */
+    public static SQLException getSQLException(int errorCode)
+    {
+        return getJdbcSQLException(errorCode, null, new String[0]);
```

Comment:
`getJdbcSQLException(errorCode, null)` is enough, third argument is not needed, compiler will generate code for an empty array by itself.